test/common.js

```diff
@@ -201,9 +202,43 @@ export function getContainer({ parent, shadow = false, slots = false } : {| pare
         }
     });
 
+    customElements.define(innerWrapperName, class extends HTMLElement {
+        connectedCallback() {
+            const shadowRoot = this.attachShadow({ mode: 'open' });
+            shadowRoot.appendChild(shadowContainer);
+        }
+    });
+
     const customElement = document.createElement(customElementName);
     parentContainer.appendChild(customElement);
 
+    if (nested) {
+        const innerWrapper = document.createElement(innerWrapperName);
+        innerWrapper.setAttribute('id', 'inner-element-wrapper-id');
+
+        const customElementShadowRoot = customElement.shadowRoot;
+        const innerWrapperShadowRoot = innerWrapper.shadowRoot;
```

Comment:
The `innerWrapper` isn't in the DOM so we can't get at it's `shadowRoot` yet. Please try this instead:

```js
if (nested) {
        const innerWrapper = document.createElement(innerWrapperName);
        innerWrapper.setAttribute('id', 'inner-element-wrapper-id');

        const customElementShadowRoot = customElement.shadowRoot;

        if (customElementShadowRoot) {
            customElementShadowRoot.appendChild(innerWrapper);
        }

        const innerWrapperShadowRoot = innerWrapper.shadowRoot;
        const innerWrapperContainer = document.createElement('div');
        innerWrapperContainer.setAttribute('id', 'super-id');

        if (innerWrapperShadowRoot) {
            innerWrapperShadowRoot.appendChild(innerWrapperContainer);
        }

        return {
            container: innerWrapperContainer,
            destroy:   () => {
                parentContainer.removeChild(customElement);
                parentContainer.removeChild(innerWrapper);
            }
        };
    }
```

Reply:
After this code change we should expect to get the following error message `Host element is also in shadow dom`:

<img width="1048" alt="expected-failure-message" src="https://user-images.githubusercontent.com/534034/118175942-63a4bf00-b3f6-11eb-8682-18af756be6a4.png">